src/App.js

```diff
@@ -1,16 +1,32 @@
 import './index.css';
+import {
+  BrowserRouter, Routes, Route, NavLink,
+} from 'react-router-dom';
 import { Calculates } from './compnents/Calculator';
-import { Quote, Footer } from './Quotes/Quote';
+import Quote from './Quotes/Quote';
+import Home from './Home/Home';
 
 function App() {
   return (
-    <>
-      <div className="main">
-        <Calculates />
-        <Quote />
-        <Footer />
-      </div>
-    </>
+    <BrowserRouter>
+      <header>
+        <nav>
+          <h1>Math Magicians</h1>
+          <div className="links">
+            <NavLink to="/">Home</NavLink>
+            <NavLink to="about">Calculator</NavLink>
+            <NavLink to="quote">Quotes</NavLink>
+          </div>
+        </nav>
```

Comment:
[OPTIONAL] Kindly consider creating a component and adding the navigation links inside, you can then import it here. This will make your project more maintanable and easy to fix bugs. :+1: